servers/dsub/jobs/test/base_test_cases.py

```diff
@@ -438,3 +453,66 @@ def test_query_jobs_submission_pagination(self):
                     page_token=response.next_page_token,
                     extensions=ExtendedQueryFields(submission=min_time)),
                 [job2])
+
+        def test_aggregtion_jobs(self):
```

Comment:
Spelling.

Also nit: would probably name this the same as the method, test_get_job_aggregations

Reply:
Accepted.